PerformanceNavigationTiming for User Agent Launch/explainer.md

```diff
@@ -50,168 +50,168 @@ secureConnectionStart: 126.19999998807907
 serverTiming: []
 startTime: 0
 transferSize: 300
-type: "navigate"  <== NavigationType
+type: "navigate"
```

Comment:
Looks to me like the spec calls this a [`NavigationTimingType`](https://w3c.github.io/navigation-timing/#dom-navigationtimingtype)